lib/Terminal/Print/Grid.pm6

```diff
@@ -44,12 +44,28 @@ method rows    { $!h }
 method height  { $!h }
 
 #| Instantiate a new (row-major) grid of size $w x $h
-method new($w, $h, :$move-cursor = move-cursor-template) {
-    my @grid = [ [ ' ' xx $w ] xx $h ];
+method new($w, $h, :@grid is copy, :$move-cursor = move-cursor-template) {
```

Comment:
Where is the fast path?  You've converted the stripped-and-fast new to a slow path.

Reply:
I don't think creating a new grid is a high-repetitive task which would make such a measurable impact on overall performance. Besides, the condition body wouldn't be executed is `@grid` is empty. Unfortunately, I don't have much time right now to bench `xx` vs. a loop, but it's unlikely that the difference would be that significant.

Reply:
Here is interesting benchmark:

```perl6
my @p =
    start {
        my $st = now;
        for ^10000 {
            my @g = [ [ ' ' xx 100 ] xx 100 ];
        }
        my $et = now;
        note "xx + xx: ", $et - $st;
    },
    start {
        my $st = now;
        for ^10000 {
            my @g;
            for ^100 -> $gr {
                my @row = ' ' xx 100;
                if False {
                    note "IMPOSSIBLE";
                }
                @g[$gr] = @row;
            }
        }
        my $et = now;
        note "for + xx: ", $et - $st;
    };

await @p;
```

And here is its output:

```
for + xx: 12.6258859
xx + xx: 14.7326081
```

The outcome is pretty much consistent.

*NOTE* I do it in parallel because consequent execution on my laptop results in CPU heating and throttling. So, second tests might display biased results. `if` simulates when no `@grid` supplied.